components/statistics/count.lua

```diff
@@ -230,6 +230,11 @@ function Count._baseConditions(args, isTournament)
 		startDateKey = 'startdate'
 		endDateKey = 'enddate'
 		sortDateKey = 'sortdate'
+
+		if not Logic.readBool(args.includeAllStatus) then
```

Comment:
this factually makes the new stuff the default so it will cause (imo) unintended changes across all wikis
so imo invert the logic here